src/layer/poi.js

```diff
@@ -260,6 +260,14 @@ var iconDefs = {
     color: Color.poi.infrastructure,
     description: "College or university",
   },
+  toilets: {
+    classes: {
+      toilets: ["toilets"],
+    },
+    sprite: "poi_toilet",
+    color: Color.poi.infrastructure,
```

Comment:
I’m not sure this fits very well with other things in the infrastructure category, or the blue category. Much of the time, it’s actually part of some other POI. Consider creating a different category for hyperlocal features that would also include street furniture. I’m assuming it would be less prominent than the infrastructure category: perhaps black, and appearing at a higher zoom level. The tradeoff would be in more remote settings where the `amenity=toilet` represents something significant in its own right. But I think that starts to get into a use case that OpenTrailMap would serve better anyways.

Reply:
> Much of the time, it’s actually part of some other POI
I don't believe places with `toilets=yes` get mapped to the `toilets` class in the OpenMapTiles schema, so the only `toilets` that should show up are POIs where the toilets are the prominent "feature" of that object, right? So assuming things are mapped "correctly" (i.e. an `amenity=toilets` is a dedicated restroom) we shouldn't run into the scenario you're describing very frequently, right? I think I'm hesitant to decrease how quickly these bathrooms/restrooms/toilets appear because I haven't seen any conflicts with other POIs so far (at least in places I'm looking).

Otherwise, I think a new category is fine, but maybe that can be a separate PR (or Issue to track that change)?